astro-client-core/mocks/client.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.20.2. DO NOT EDIT.
+// Code generated by mockery v2.16.0. DO NOT EDIT.
```

Comment:
nit: might want to update your local mockery version

Reply:
I see other files are using 2.16.0. Ok for us to upgrade to using v2.20.2 overall?

Reply:
thanks ❤️